app/plugins/Wallet/js/buttons.js

```diff
@@ -11,6 +11,22 @@ eID('create-address').onclick = function() {
 	IPC.sendToHost('api-call', call, 'new-address');
 };
 
+// Filter address list by search string
+eID('search-bar').onkeydown = function() {
```

Comment:
need to use `onkeyup` here; otherwise `searchstr` will contain the text in the search bar *prior* to the keypress.